README.rst

```diff
@@ -13,8 +13,8 @@ We'd love for you to `contribute to Girder <CONTRIBUTING.md>`_.
 
 .. |logo| image:: clients/web/static/img/Girder_Favicon.png
 
-.. |build-status| image:: https://travis-ci.org/girder/girder.svg?branch=master
-    :target: https://travis-ci.org/girder/girder
+.. |build-status| image:: https://circleci.com/gh/girder/girder.png?style=shield
```

Comment:
Yeah, the last PR I merged had a spurious error.  That will happen from time to time.  I didn't rerun it this time in case someone wanted to look at it.